### ISSUE TITLE
Bro changed star number 

### ISSUE THREAD

**Issue by xXprogtXx1** [clipssender31/Discord-Token-Generator#92]
Bruh why did you changed star obiettive, you had asked 110 and that was a lot, now you asking for 150, then you will asks for 200? Maybe 250?

**Comment by lkeld** [comment]
Yes, hes just going to keep changing it lmao hes never going to actually release it.

**Comment by QuartzWarrior** [comment]
:skull:
